app/include/rtc/rtctime_internal.h

```diff
@@ -187,8 +187,9 @@
 #define CPU_BOOTUP_MHZ    52
 
 #ifdef RTC_DEBUG_ENABLED
+#warn this does not really work....
```

Comment:
Debugging this code is a nightmare as some of it runs before the BSS segment is initialized and you can't print anything...